main/university/models.py

```diff
@@ -0,0 +1,63 @@
+from django.db import models
+
+class Faculty(models.Model):
+    name = models.CharField(max_length=200, unique=True)
+    description = models.TextField(blank=True, null=True)
+
+    class Meta:
+        verbose_name = "Faculty"
+        verbose_name_plural = "Faculties"
+
+    def __str__(self):
+        return self.name
+
+
+class Position(models.Model):
+    FULL = 'full'
+    HALF = 'half'
+    THREE_QUARTER = '3/4'
```

Comment:
non consistent naming: 3/4 -> three-quarter, but I still think it should be integer instead